src/Auxiliary/Aux_ComputeProfile.cpp

```diff
@@ -626,10 +626,13 @@ void Aux_ComputeProfile( Profile_t *Prof[], const double Center[], const double
 //       skip empty bins since both their data and weight are zero
          if ( Prof[p]->NCell[b] > 0L )
          {
-            Prof[p]->Data[b] /= Prof[p]->Weight[b];
-            Prof[p]->Data_Sigma[b] = Prof[p]->Data_Sigma[b]/Prof[p]->Weight[b] - Prof[p]->Data[b]*Prof[p]->Data[b];
-            if (Prof[p]->Data_Sigma[b] < 0.0)   Prof[p]->Data_Sigma[b] = 0.0;
-            else                                Prof[p]->Data_Sigma[b] = sqrt(Prof[p]->Data_Sigma[b]);
+            Prof[p]->Data      [b] /= Prof[p]->Weight[b];
+            Prof[p]->Data_Sigma[b]  = Prof[p]->Data_Sigma[b]/Prof[p]->Weight[b] - SQR( Prof[p]->Data[b] );
+
+            if ( Prof[p]->Data_Sigma[b] < 0.0 )
+               Aux_Error( ERROR_INFO, "Prof[%d]->Data_Sigma[%d] = %14.7e < 0.0 !!\n", p, b, Prof[p]->Data_Sigma[b] );
+            else
+               Prof[p]->Data_Sigma[b] = sqrt(Prof[p]->Data_Sigma[b]);
```

Comment:
Add one more space before and after `Prof[p]->Data_Sigma[b]` to make it more readable.